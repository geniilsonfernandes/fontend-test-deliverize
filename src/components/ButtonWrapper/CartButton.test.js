import { screen } from "@testing-library/react";
import { ButtonWrapper } from ".";
import renderWithTheme from "../../utils/test/renderWithTheme";

describe("<ButtonWrapper />", () => {
  it("should render current place in the document", () => {
    const { container } = renderWithTheme(<ButtonWrapper>foo</ButtonWrapper>);

    expect(screen.getByText("foo")).toBeInTheDocument();
    expect(container).toMatchSnapshot();
  });
});
