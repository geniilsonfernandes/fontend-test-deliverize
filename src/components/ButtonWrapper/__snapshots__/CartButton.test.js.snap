// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ButtonWrapper /> should render current place in the document 1`] = `
<div>
  <button
    class="sc-bczRLJ bFvGUo"
  >
    foo
  </button>
</div>
`;
